evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -680,6 +728,52 @@ def sum(y_true, y_pred):
         assert best_worst_report == answer
 
 
+def test_explain_predictions_time_series(ts_data):
+    X, y = ts_data
+
+    class TSPipeline(TimeSeriesRegressionPipeline):
+        component_graph = ["Delayed Feature Transformer", "Random Forest Regressor"]
+        name = "time series pipeline"
+
+    tspipeline = TSPipeline({"pipeline": {"gap": 1, "max_delay": 2}})
+
+    tspipeline.fit(X, y)
+
+    exp = explain_predictions(pipeline=tspipeline, input_features=X, y=y,
+                              indices_to_explain=[5, 11], output_format="dict")
+
+    # Check that the computed features to be explained aren't NaN.
+    for exp_idx in range(len(exp["explanations"])):
+        assert not np.isnan(np.array(exp["explanations"][exp_idx]["explanations"][0]["feature_values"])).any()
+
+    with pytest.raises(ValueError, match="Requested index"):
+        explain_predictions(pipeline=tspipeline, input_features=X, y=y,
+                            indices_to_explain=[1, 11], output_format="text")
+
+
+@pytest.mark.parametrize("pipeline_class, estimator", [(TimeSeriesRegressionPipeline, "Random Forest Regressor"), (TimeSeriesBinaryClassificationPipeline, "Logistic Regression Classifier")])
+def test_explain_predictions_best_worst_time_series(pipeline_class, estimator, ts_data):
+    X, y = ts_data
+
+    if is_binary(pipeline_class.problem_type):
+        y = y % 2
+
+    class TSPipeline(pipeline_class):
+        component_graph = ["Delayed Feature Transformer", estimator]
+        name = "time series pipeline"
+
+    tspipeline = TSPipeline({"pipeline": {"gap": 1, "max_delay": 2}})
+
+    tspipeline.fit(X, y)
+
+    exp = explain_predictions_best_worst(pipeline=tspipeline, input_features=X, y_true=y,
+                                         output_format="dict")
+
+    # Check that the computed features to be explained aren't NaN.
+    for exp_idx in range(len(exp["explanations"])):
+        assert not np.isnan(np.array(exp["explanations"][exp_idx]["explanations"][0]["feature_values"])).any()
```

Comment:
We're not checking the output against a specified value. The other tests check that the logic for selecting the rows that are explained and displaying the right features/shap values is correct so I think running real data through and verifying it completes without showing NaNs is sufficient.